lib/benchmarks/initial-render.bench.tsx

```diff
@@ -113,6 +114,32 @@ function ReactHookFormInitialRenderBenchmark() {
   );
 }
 
+function ReactFinalFormInitialRenderBenchmark() {
+
+  return (
+    <Form
+      onSubmit={onSubmit}
+      validate={validate}
+      render={({ handleSubmit, name }) => (
+        <form onSubmit={handleSubmit}>
+          {arr.map((num, i) => (
+            <Field
+              name={name}
+              render={({ input, meta }) => (
+                <div>
+                  <label>{name}</label>
+                  <textarea {...input} />
+                  {meta.touched && meta.error && <span>{meta.error}</span>}
+                </div>
```

Comment:
This does not match 1:1 the other benchmarks. Remember, the point of these benchmarks is to make sure that HouseForm and other libraries are doing _the same things_ to compare as closely as we can their performance.

Let's remove the `div`, `textarea`, and `span` entirely, since the other benchmarks aren't doing this.